examples/suspend-template.yaml

```diff
@@ -18,12 +21,18 @@ spec:
         template: whalesay
     - - name: approve
         template: approve
+    - - name: delay
+        template: delay
     - - name: release
         template: whalesay
 
   - name: approve
     suspend: {}
 
+  - name: delay
+    suspend:
+      resumeAfterSeconds: 20
```

Comment:
I think `duration` or `wakeup` will fit for this functionality. @jessesuen can comment

Reply:
`duration` is good. Maybe `durationSeconds`?

Reply:
`duration` is good.